src/web_resources/HangarXPLOR.BulkUI.js

```diff
@@ -38,8 +39,6 @@ HangarXPLOR.BulkUI = function()
   HangarXPLOR.$bulkUI.$downloads = $('<div>', { class: 'actions' });
   HangarXPLOR.$bulkUI.$loading = $('<div>', { class: 'status value' });
   
-  HangarXPLOR.$bulkUI.addClass(HangarXPLOR._feature.Summary);
-  
```

Comment:
You removed the BulkXPLOR hooks! :'(

Reply:
This was awhile ago; I don't remember why; feel free to undo